DANS-KNAW/easy-stage-dataset
# EASY-1171: extra parameter voor stage-dataset - stubAVfiles

fixes EASY-1171 extra parameter voor stage-dataset: stubAVfiles

#### When applied it will
* When stubAVfiles is true all files that are of type Audio or Video will be 'stubbed' and the filedata is not stored in the Fedora archive.
 
#### Where should the reviewer @DANS-KNAW/easy start?
src/main/scala/nl/knaw/dans/easy/stage/Conf.scala
and
src/main/scala/nl/knaw/dans/easy/stage/EasyStageDataset.scala

#### How should this be manually tested?
* build with mvn clean install
* deploy (on deasy) with deploy-role easy_stage_dataset
* use the 'minimal-av' test dataset and run easy-stage-dataset, to get it into EASY also run easy-ingest and easy-update-fs-rdb and easy-update-solr-index ...

More practical is to run the unit tests, which should succeed, and then inspect the result for the av test dataset in: target/sdoPuddings/minimal-av
